Mobile/ShopLens/Droid/Helpers/ShopLensTextToSpeech.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using Android.Content;
+using Android.Runtime;
+using Android.Speech.Tts;
+using Java.Util;
+
+namespace ShopLens.Droid.Helpers
+{
+    class ShopLensTextToSpeech : Java.Lang.Object, TextToSpeech.IOnInitListener
+    {
+        event EventHandler TTSInitialized;
+        ShopLensUtteranceProgressListener uttListener;
+
+        public TextToSpeech tts;
```

Comment:
If `ShopLensTextToSpeech` is a wrapper, maybe make this field private and expose only the properties you need? So far I see `tts.IsSpeaking` is used elsewhere